src/ucp/wireup/wireup_ep.h

```diff
@@ -30,15 +30,20 @@ enum {
  * endpoint it also contains an auxiliary endpoint which can send wireup messages.
  */
 struct ucp_wireup_ep {
-    ucp_proxy_ep_t            super;        /**< Derive from ucp_proxy_ep_t */
+    ucp_proxy_ep_t            super;         /**< Derive from ucp_proxy_ep_t */
     ucs_queue_head_t          pending_q;     /**< Queue of pending operations */
     uct_ep_h                  aux_ep;        /**< Used to wireup the "real" endpoint */
+    uct_ep_h                  sockaddr_ep;   /**< Used for client-server wireup */
     ucp_rsc_index_t           aux_rsc_index; /**< Index of auxiliary transport */
     volatile uint32_t         pending_count; /**< Number of pending wireup operations */
     volatile uint32_t         flags;         /**< Connection state flags */
     uct_worker_cb_id_t        progress_id;   /**< ID of progress function */
 };
 
+typedef struct ucp_wireup_client_data {
+    ucp_err_handling_mode_t err_mode;
+    uint64_t                ep_uuid;
```

Comment:
could you please add here that /* packed worker address follows */ ?

Reply:
ok